Tests.Unit.Core.ApplicationServices/ApplicationServices/GDPR/DataProcessingRegistrationApplicationServiceTest.cs

```diff
@@ -1229,6 +1229,108 @@ public void Cannot_ClearBasisForTransfer_If_Write_Access_Is_Denied()
             Test_Command_Which_Fails_With_Dpr_Insufficient_WriteAccess(id => _sut.ClearBasisForTransfer(id));
         }
 
+        [Fact]
+        public void Can_Update_IsOversightCompleted()
+        {
+            //Arrange
```

Comment:
Prøv og se hvordan de andre Success cases er lavet. Der er en hjælpe metode for at mindske kode duplikation